spec/controllers/admin/reports/registered_adviser_controller_spec.rb

```diff
@@ -0,0 +1,35 @@
+RSpec.describe Admin::Reports::RegisteredAdviserController, type: :controller do
+  describe '#filename' do
+    before(:each) do
+      Timecop.freeze(date)
+    end
+
+    after(:each) do
+      Timecop.return
+    end
+
+    context 'yesterday' do
```

Comment:
♻️ @norm && @willhall88 : how about this rather terse context definitions here https://github.com/moneyadviceservice/rad/pull/406/commits/2e87d92add386ddb66f7b06d4df3b1e4a04491da. Do they communicate the context of these tests well enough or is there further room for improvement?